client/components/cards/cardDetails.jade

```diff
@@ -143,85 +172,85 @@ template(name="cardDetails")
     //- XXX We should use "editable" to avoid repetiting ourselves
     if canModifyCard
       unless currentUser.isWorker
-        h3
-          i.fa.fa-align-left
-          card-details-item-title {{_ 'description'}}
+        //h3
+          //- i.fa.fa-align-left
+          //- card-details-item-title {{_ 'description'}}
         +inlinedCardDescription(classNames="card-description js-card-description")
           +editor(autofocus=true)
             | {{getUnsavedValue 'cardDescription' _id getDescription}}
           .edit-controls.clearfix
             button.primary(type="submit") {{_ 'save'}}
             a.fa.fa-times-thin.js-close-inlined-form
         else
-          a.js-open-inlined-form
+          a.description-item.add-description.js-open-inlined-form
             if getDescription
               +viewer
                 = getDescription
             else
-              | {{_ 'edit'}}
+              | {{_ 'Add a more detailed description'}}
```

Comment:
You have not yet added this translation also to `wekan/i18n/en.i18n.json`.

Reply:
Although I don't know why this should be changed?

Reply:
I think this change should be removed.

Reply:
I will fix it